prompt-service/src/unstract/prompt_service/config.py

```diff
@@ -62,6 +65,9 @@ def create_app() -> Flask:
         password=db_pass,
         host=db_host,
         port=db_port,
+        max_connections=max_connections,
+        stale_timeout=stale_timeout,
+        timeout=timeout,
     )
     db.connect()
```

Comment:
@ritwik-g It is not a big change. Just 2 additional lines.

Reply:
Even I believe the way @Deepak-Kesavan  suggested is a better approach.
I will go ahead and merge as it has two approvals. @harini-venkataraman  can you take a look at @Deepak-Kesavan  suggestions and make the change?

Reply:
@Deepak-Kesavan Will address it in a separate PR. Though prompt service is straight forward, need some thorough testing in platform service, since we initialize db in a different file and create app in run.